heartbeat_publisher:
  ip_address:
    type: string
    default_value: "255.255.255.255"
    description: "IP address of the emergency stop board."
  port:
    type: int
    default_value: 1000
    description: "Port of the emergency stop board."
    validation:
      gt<>: 0
  from_port:
    type: int
    default_value: 1000
    description: "Port of the control PC."
    validation:
      gt<>: 0
  publish_rate:
    type: double
    default_value: 10.0
    description: publish rate of protolink message
    validation:
      gt<>: 0
